integrations/user_test.go

```diff
@@ -92,7 +92,7 @@ func TestRenameReservedUsername(t *testing.T) {
 		htmlDoc := NewHTMLParser(t, resp.Body)
 		assert.Contains(t,
 			htmlDoc.doc.Find(".ui.negative.message").Text(),
-			i18n.Tr("en", "user.newName_reserved"),
+			i18n.Tr("en", "form.name_reserved", reservedUsername),
```

Comment:
Should be `i18n.Tr("en", "user.form.name_reserved", reservedUsername)`
You just forget the user.

Reply:
I did exactly that, but it fails vecause `i18n.Tr` doesn't parse the `%s` format specifier in the translation string. See the CI report.

Reply:
No you write "form.name_reserved" not "user.form.name_reserved"

Reply:
If it can find the string in translations it will default to the trad name so in your case name_reserved or previously newName_reserved and add all extra string that are not in the translated format after.

Reply:
@sapk Thanks for the coaching, I've updated my branch again. Waiting for the CI now.